integration/client-cli/admin-cli/src/main/bin/kcadm.bat

```diff
@@ -5,4 +5,24 @@ if "%OS%" == "Windows_NT" (
 ) else (
   set DIRNAME=.\
 )
-java %KC_OPTS% -cp "%DIRNAME%\client\keycloak-admin-cli-${project.version}.jar" --add-opens=java.base/java.security=ALL-UNNAMED -Dkc.lib.dir="%DIRNAME%\client\lib" org.keycloak.client.admin.cli.KcAdmMain %*
+
+if "x%JAVA_HOME%" == "x" (
```

Comment:
The check from the sh files:

```
if [ -z "$JAVA" ]; then
    if [ -n "$JAVA_HOME" ]; then
        JAVA="$JAVA_HOME/bin/java"
    else
        JAVA="java"
    fi
fi
```
That implies there should be an outer check here for JAVA:
```
if "x%JAVA%" == "x" (
  if "x%JAVA_HOME%" == "x" (
    set JAVA=java
    :: should this be on stdout and/or something like this be added to the .sh scripts?
    echo JAVA_HOME is not set. Unexpected results may occur.
    echo Set JAVA_HOME to the directory of your local JDK to avoid this message.
  ) else (
    ...
  )
)
```